msteams-platform/resources/team-chat-member-api-changes.md

```diff
@@ -47,9 +52,13 @@ Here’s a sample JSON representation of what’s returned by these APIs today.
 ## API Changes
 Here are the upcoming API changes:
 
-* We've created a new API [`TeamsInfo.GetPagedMembersAsync`](https://docs.microsoft.com/microsoftteams/platform/bots/how-to/get-teams-context?tabs=dotnet#fetching-the-roster-or-user-profile) for retrieving profile information for members of a chat/team. This API is now available with the Bot Framework 4.8 SDK. For development in all other versions use the [`GetConversationPagedMembers`](https://docs.microsoft.com/dotnet/api/microsoft.bot.connector.conversationsextensions.getconversationpagedmembersasync?view=botbuilder-dotnet-stable) method. **Note**: In either v3 or v4, the best action is to upgrade to the latest point release (3.30.2 or 4.8 respectively). 
-* We've created a new API [`TeamsInfo.GetMemberAsync`](https://docs.microsoft.com/microsoftteams/platform/bots/how-to/get-teams-context?tabs=dotnet#get-single-member-details) for retrieving the profile information for a single user. It takes the ID of the team/chat and a [UPN](https://docs.microsoft.com/windows/win32/ad/naming-properties#userprincipalname) (`userPrincipalName`, *see above*), Azure Active Directory Object ID (`objectId`, *see above*), or the Teams user ID (`id`, *see above*) as parameters and returns the profile information for that user. **Note**: We're changing `objectId` to `aadObjectId` to match what it's called in the `Activity` object of a Bot Framework message. The new API is available with version 4.8 of the Bot Framework SDK. It will soon be available in the Teams SDK extension Bot Framework 3.x as well; meanwhile you can use the [REST](https://docs.microsoft.com/microsoftteams/platform/bots/how-to/get-teams-context?tabs=json#get-single-member-details) endpoint.
-* `TeamsInfo.GetMembersAsync` (C#) and `TeamsInfo.getMembers` (TypeScript/Node.js) is formally deprecated and will stop working in late 2021. We'll announce a specific timetable in May 2020, based on feedback from developers. Once the new paged API is available, developers should update their bots to use it. (This also applies to the [underlying REST API these APIs use](https://docs.microsoft.com/microsoftteams/platform/bots/how-to/get-teams-context?tabs=json#tabpanel_CeZOj-G++Q_json).)
+* We've created a new API [`TeamsInfo.GetPagedMembersAsync`](../../bots/how-to/get-teams-context.md?tabs=dotnet#fetching-the-roster-or-user-profile) for retrieving profile information for members of a chat/team. This API is now available with the Bot Framework 4.10 SDK. For development in all other versions use the [`GetConversationPagedMembers`](/dotnet/api/microsoft.bot.connector.conversationsextensions.getconversationpagedmembersasync?view=botbuilder-dotnet-stable&preserve-view=true) method. 
+  > [!NOTE]
+  > In either v3 or v4, the best action is to upgrade to the latest point release (3.30.2 or 4.10 respectively). 
```

Comment:
Not clear to me exactly what we mean here. (4.10 isn't the latest release of the SDK.)

Reply:
@aosolis - can we remove the note then? This was already there in the document, I have just formatted it as a note as we do in markdown.

Reply:
removing mention of version to simplify